README.md

```diff
@@ -3,6 +3,22 @@
 ## FAQ
 [click here](/docs/faq.md)
 
+# Configurations
+
+### Override Images:
+Following environment variables allows overriding images of pipelines, trigger and 
+`Task` addons components
+
+1) `IMAGE_PIPELINE_<deployment-image-name>` e.g. `IMAGE_PIPELINE_WEBHOOK`
+It allows overriding pipelines or triggers deployment images. Note, `*_PIPELINE_*` 
+will override images in pipelines manifest only. Same way `*_TRIGGERS_*`
+2) `IMAGE_PIPELINE_<deployment-image-arg-name>` e.g. `IMAGE_PIPELINE_-NOP`
```

Comment:
is `IMAGE_PIPELINE_-NOP` a valid shell variable?

Reply:
right, felt so as well, shall convert to `_ARG_` or assume replacement should look for image `name` or `arg`?